unicorn/src/cluster/unicorn.cpp

```diff
@@ -244,6 +244,10 @@ void unicorn_cluster_t::on_node_list_change(size_t scope_id, std::future<respons
                 endpoint_map.erase(to_delete[i]);
             }
             for (auto& uuid: nodes) {
+                if(uuid == locator.uuid()) {
```

Comment:
Fixed a bug related to announcing local service as remote. That was important for vicodyn, but got here by mistake